_pages/contributing/updating-schematics.md

```diff
@@ -1,42 +1,74 @@
+
+## Migrations Mechanism
+
+After upgrading to a new major version, the migration mechanism should be updated at the very beginning of the new development cycle. For example, if Spartacus has been updated from version 2.x to 3.0, the updated mechanism should be merged to the `develop` branch as soon as possible. This allows contributors to include migrations with their features and bug fixes from the very start of the development cycle.
 
 ## Structure for Updating Schematics
 
-The `projects/schematics/src/migrations/migrations.json` file contains all migration scripts for all Spartacus versions:
+The `projects/schematics/src/migrations/migrations.json` file contains a list of all the migration scripts for every Spartacus version. The following is an example of a migration script:
+
+```json
+"migration-v3-constructor-deprecations-03": {
+      "version": "3.0.0",
+      "factory": "./3_0/constructor-deprecations/constructor-deprecations#migrate",
+      "description": "Add or remove constructor parameters"
+    },
+```
+
+Each script has a set of properties, which are described as follows:
 
-- _name_ property is important for developers to quickly understand what the migration script is doing. By convention, the migration _name_ should follow `migration-v<version>-<migration-feature-name>-<sequence-number>` pattern, where:
-  - _version_ should indicate for which Spartacus version the migration is intended.
-  - _migration-feature-name_ is a short name that describes what the migration is doing.
-  - _sequence-number_ is the sequence number in which the migrations should be executed
-  - An example is _migration-v2-update-cms-component-state-02_.
-- _version_ is _really_ important for the Angular's update mechanism, as it is used to automatically execute the required migration scripts for the current project's version. For more information about this, please check [releasing update schematics](#releasing-update-schematics) section.
-- _factory_ - points to the specific migration script.
-- _description_ - a short free-form description field for developers.
+- `name` allows developers to quickly understand what the migration script is doing. The migration `name` has the following pattern: `migration-v<version>-<migration-feature-name>-<sequence-number>`. The elements of `name` are as follows:
+  - `version` indicates which version of Spartacus the migration is intended for.
+  - `migration-feature-name` is a short name that describes what the migration is doing.
+  - `sequence-number` indicates the order of execution for the migration scripts. For example, if a script has a `sequence-number` of `03`, it will be the third script to execute when the migration scripts are run.
+- `version` is very important for the Angular update mechanism. It is used to automatically run the required migration scripts for a specific version. For more information, see the [Releasing Update Schematics](#releasing-update-schematics) section.
```

Comment:
The link here seems to be pointing to a section that does not exist.

Reply:
should be this one: https://github.com/SAP/spartacus/tree/develop/projects/schematics#releasing-update-schematics